assets/helpers/csrf/csrfReducer.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Action } from './csrfActions';
+
+
+// ----- Types ----- //
+
+type Csrf = {
+    token: ?string,
+};
+
+
+// ----- Setup ----- //
+
+const initialState: Csrf = window.guardian && window.guardian.csrf ? {
```

Comment:
Hmm, @davidfurey were you basing this on what I did for the [user reducer](https://github.com/guardian/support-frontend/blob/master/assets/helpers/user/userReducer.js#L21)? Looking at what I did there I think I agree with @svillafe, i.e. that wasn't the best solution.

What I should have done is passed this through to the `createStore` function as the `preloadedState`. The [docs](http://redux.js.org/docs/recipes/reducers/InitializingState.html) describe this parameter as being used for 'hydrating the store from some persistent storage or the server', which I think is what we're doing both here and in the case of the user reducer.